pkg/features/kube_features.go

```diff
@@ -622,7 +622,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	PodOverhead:                                    {Default: true, PreRelease: featuregate.Beta},
 	InPlacePodVerticalScaling:                      {Default: false, PreRelease: featuregate.Alpha},
 	PerNetworkServiceIPAlloc:                       {Default: false, PreRelease: featuregate.Alpha},
-	MandatoryArktosNetwork:                         {Default: false, PreRelease: featuregate.Alpha},
+	MandatoryArktosNetwork:                         {Default: true, PreRelease: featuregate.Beta},
```

Comment:
will this change from false to true break kubemark testing?

Reply:
Yes, will change back to false. Then instead to  start kubelet with --feature-gates=true

Reply:
The correct way is use environment setting to enable feature MandatoryArktosNetwork to be true in shell command   - export FEATURE_GATES="AllAlpha=false,MandatoryArktosNetwork=true"